File: Julia/8kyu/8kyu-count-odd-numbers-below-n.jl
# Count Odd Numbers below n
# Language: Julia
# Version: 1.5
# Difficulty: 8 kyu

module Solution
  export oddcount
  
  function oddcount(n)
    floor(n / 2)
  end
  
end


# Tests
using FactCheck
using .Solution

facts("Sample Tests") do
  context("Basic Tests") do
     @fact oddcount(15) --> 7
     @fact oddcount(15023) --> 7511
  end
end

File: R/8kyu/8kyu-count-odd-numbers-below-n.r
# Count Odd Numbers below n
# Language: R
# Version: 3.4.1
# Difficulty: 8 kyu

odd_count <- function(n) {
  n %/% 2
}


# Tests
test_that("Sample tests", {
  expect_equal(odd_count(15), 7)
  expect_equal(odd_count(15023), 7511)
})